lark/load_grammar.py

```diff
@@ -426,9 +423,9 @@ def _literal_to_pattern(literal):
 
     if literal.type == 'STRING':
         s = s.replace('\\\\', '\\')
-        return PatternStr(s, flags)
+        return PatternStr(s, flags, raw=literal.value)
```

Comment:
Is it really necessary to save the original value?

Reply:
Do you have a better idea? The goal is to show exactly this.